file/validate.go

```diff
@@ -31,3 +31,11 @@ func validate(content []byte) error {
 	}
 	return errs
 }
+
+func validateWorkspaces(workspaces []string) error {
+	utils.RemoveDuplicates(&workspaces)
+	if len(workspaces) > 1 {
+		return fmt.Errorf("cannot sync multiple workspaces at the same time: %v", workspaces)
```

Comment:
```suggestion
		return fmt.Errorf("It seems like you are trying to sync multiple workspaces at the same time (%v). decK doesn't support syncing multiple workspaces at the same time, please sync one workspace at a time.", workspaces)
```

Use appropriate \ns and maybe add an HTTP link to docs if we have one.
cc @lena-larionova

Reply:
The doc is light on this, needs some improvement, but we do have something: https://docs.konghq.com/deck/1.10.x/guides/kong-enterprise/#workspaces